src/typings.ts

```diff
@@ -15,6 +15,7 @@ export interface IClusterMapProps extends MapViewProps {
   onMapReady: () => void;
   onClusterClick: (clusterClickEvent?: IClusterClickEvent) => void;
   onRegionChangeComplete: (region: Region) => void;
+  customMarkerProps?: object;
```

Comment:
Is this is a prop only for ClusterMarker component, let's call it `clusterMarkerProps?: object`

Reply:
Hey! Thank you for the lib and such a quick reply, that I did not expect 👍 
I was in a hurry and did not think a lot about naming... my bad.